subt_ign/src/CommsBrokerPlugin.cc

```diff
@@ -324,8 +324,11 @@ void CommsBrokerPlugin::UpdateIfNewBreadcrumbs()
   if (newBreadcrumbFound)
   {
     std::set<ignition::math::Vector3d> breadcrumbPoses;
+    #pragma GCC diagnostic push
+    #pragma GCC diagnostic ignored "-Wunused-variable"
```

Comment:
If we aren't using the variable, I would prefer that we update the code, rather than suppressing the warning.

We should be able to do 

```
for (const auto& it: this->breadcrumbs)
{
  breadcrumbPoses.insert(it->second.Pos());
}
```

Reply:
thanks! changed.